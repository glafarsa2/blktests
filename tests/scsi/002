#!/bin/bash
# SPDX-License-Identifier: GPL-3.0+
# Copyright (C) 2017 Johannes Thumshirn
#
# Regression test for commit 68c59fcea1f2 ("scsi: sg: fix SG_DXFER_FROM_DEV
# transfers").

. tests/scsi/rc

DESCRIPTION="perfom a SG_DXFER_FROM_DEV from the /dev/sg read-write interface"
QUICK=1

requires() {
	_have_scsi_generic && _have_src_program sg/dxfer-from-dev
}

test_device() {
	echo "Running ${TEST_NAME}"

	SG_DEV="/dev/$(_get_test_dev_sg)"
	"$SRCDIR"/sg/dxfer-from-dev "$SG_DEV"

	echo "Test complete"
}
