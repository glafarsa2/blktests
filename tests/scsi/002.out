Running scsi/002
PASS
Test complete
